UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -825,9 +840,8 @@ <h4>Instructions</h4>
 							</li>
 							<li><b>LET</b> <var>hazards_string</var> be the result of:
 								<ul class="condition">
-									<li>calling join on <var>hazards</var> with the separator <code>", "</code></li>
-									<li>making uppercase the first character</li>
-									<li>replacing the last occurrence of <code>", "</code> with <code id="join-array-and">", and "</code></li>
+									<li>calling <a href="#join-array-to-comma-list">join array to comma list</a> given <var>hazards</var></li>
+									<li>uppercasing the first character</li>
```

Comment:
Here too talks about uppercasing the first character.

Reply:
That's right, it is a sub-level list, where in the variable assignment the array is merged and the first letter is capitalized, this is a mode we already had before the introduction of the new function and was suggested by @mattgarrish

For example:
<ol><li><b>LET</b> <var>hazards_string</var> be the result of:
								<ul class="condition">
									<li>calling <a href="#join-array-to-comma-list">join array to comma list</a> given <var>hazards</var></li>
									<li>uppercasing the first character</li>
									<li>concatenating <code id="hazards-plural">" hazards"</code> to the end of the string <b>IF</b> (length of <var>hazards</var>) > 1 <b>ELSE</b> concatenating <code id="hazards-singular">" hazard"</code> to the end of the string.</li>
								</ul>
							</li></ol>

Reply:
I cannot insert uppercasing within the `join array to comma list` function, because the generated string does not always have to be capitalized at the beginning.

Reply:
Could you explain what your concern is, Charles? Uppercase functions are pretty ubiquitous, and almost never manually coded, so I don't expect there's any value in documenting how that works.

There's probably an argument that uppercasing isn't applicable to every language, but I don't know that that needs stating. I'm sure anyone implementing a display transform for such languages will already understand that.

Reply:
I don't care if we capitalize or not, my concern is that in the code it says this function will capitalize the first word which it doesn't.

Reply:
It doesn't say that the function will capitalize. It's a separate list item, so a separate instruction to whoever is implementing this step to uppercase the first letter.

Would it be clearer for you if it said "Uppercasing the first character in the resulting string"?